packages/server/src/util/test-utils/user.test-generator.ts

```diff
@@ -0,0 +1,14 @@
+import { User } from '../../api/user/user.model';
+import { uniqueEdipi, uniquePhone } from './unique';
+
+export function getAdminUserTestData() {
```

Comment:
We should probably parameterize this method more like the others.

Reply:
Yes, but out of the scope for now.